tests/test.bkl

```diff
@@ -298,6 +298,38 @@
         <wx-lib>base</wx-lib>
     </exe>
 
+    <exe id="test_allheaders" template="wx_sample,wx_test"
+                              template_append="wx_append"
+         cond="USE_GUI=='1'">
+
+        <!-- link against GUI libraries, but be a console app: -->
+        <app-type>console</app-type>
+
+        <sources>
+            asserthelper.cpp
+            test.cpp
+            allheaders.cpp
+            testableframe.cpp
+        </sources>
+        <!--
+            Can't use <wx-lib> here as it doesn't work with conditionally
+            defined variables, so fall back <sys-lib> as we must not link with
+            inexisting webview library to be able to run tests even if
+            wxWebView or wxSTC is not available.
+         -->
+        <sys-lib>$(WXLIB_WEBVIEW)</sys-lib>
+        <sys-lib>$(WXLIB_STC)</sys-lib>
+        <wx-lib>aui</wx-lib>
+        <wx-lib>richtext</wx-lib>
+        <wx-lib>media</wx-lib>
+        <wx-lib>xrc</wx-lib>
+        <wx-lib>xml</wx-lib>
+        <wx-lib>html</wx-lib>
```

Comment:
Do we really need all those? I'm surprised that this test actually references symbols from them...